pypesto/__init__.py

```diff
@@ -47,7 +47,9 @@
     AdaptiveMetropolisSampler,
     ParallelTemperingSampler,
     AdaptiveParallelTemperingSampler,
-    McmcPtResult)
+    Pymc3Sampler,
+    McmcPtResult,
+    geweke_test)
```

Comment:
why added here the `geweke_test` function?

Reply:
to be able to call it shorter as `pypesto.geweke_test`

Reply:
I see that, but does it make sense? for me personally makes more sense and helps with the structure to have it inside `diagnostics`, same for the autocorrelation once is finished.

Reply:
alternative would be calling as `pypesto.sampling.geweke_test` (currently). `pypesto.sampling.diagnostics.geweke_test` is a bit long I think ...

Reply:
yes the first one i think it is good, then you somehow show that is sampling related
what do you think?

Reply:
Problem is that currently, all the other sampling module thingies can be called from pypesto namespace. So, either we should force to call all of them as `pypesto.sampling.xyz`, or all/most as `pypesto.xzy`, I think. Both would be fine for me, maybe to be discussed separately.

Reply:
sorry, did not see your last comment. yes, let's discuss separately.

Reply:
see #355